src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(BallsCollors.values().length);
```

Comment:
Creating a new instance of Random every time the method is called can be inefficient. Consider creating a single instance of Random as a class-level field and reusing it here.